addons/binding/org.openhab.binding.helios/ESH-INF/thing/ipvario213.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="helios"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="ipvario213">
+		<label>Helios IP Vario</label>
+		<description>Helios IP Vario Door Station / Intercom with Firmware v2.13</description>
+
+		<channels>
+			<channel id="keypressed" typeId="keypressed" />
+			<channel id="keypressedstamp" typeId="timestamp" />
```

Comment:
Could you create individual channel types for the different time stamps (you then also have the chance to provide different labels for them).